services/api-server/src/simcore_service_api_server/models/schemas/model_adapter.py

```diff
@@ -121,3 +125,20 @@ class ServicePricingPlanGetLegacy(BaseModel):
 assert set(ServicePricingPlanGetLegacy.model_fields.keys()) == set(
     _ServicePricingPlanGet.model_fields.keys()
 )
+
+
+class LicensedItemGet(BaseModel):
```

Comment:
It would be good to have base classes to identify whether this is input or output. Even if they are basically typealiases. It reads much better having something like

```python

OutputSchema: TypeAlias = BaseModel
InputSchema: TypeAlias = BaseModel

class BookGet(OutputSchema):
      id: str
      title: str
      author: str
      
     @classmethod
     def from_domain_model(cls, b: Book) -> Self:
           # domain to schema model adapter

class BookPatch(InputSchema):
      title: str | None = None
      author: str | None = None

      def to_domain_model(self) -> BookUpdate:
           # schema to domain model adapter
    
 
DomainModel: TypeAlias = BaseModel

 class Book(DomainModel):
       ...
      
  class BookUpdate(DomainModel):
      ...       
```